texar/modules/decoders/decoder_helpers.py

```diff
@@ -518,6 +537,69 @@ def sample(self, time: int, outputs: torch.Tensor) -> torch.LongTensor:
         return sample_ids
 
 
+class TopPSampleEmbeddingHelper(SingleEmbeddingHelper):
+    r"""A helper for use during inference.
+
+    Samples from candidates that have a cumulative probability of at most `p`
+    when arranged in decreasing order, and passes the result through an
+    embedding layer to get the next input.
+
```

Comment:
Add the paper reference to the doc, sth like:

`This is also named as "Nucleus Sampling" as proposed in the paper (Holtzman et al) "The Curious Case of Neural Text Degeneration"`

Reply:
Resolved in d07e236